Standards/CCSMB-X.md

```diff
@@ -0,0 +1,38 @@
+# WAV format for DFPWM data
+*Author: JackMacWindows <@MCJack123>*
+*Version: v1.0.0*
+*Last updated: 2022-05-02*
+
+This document describes how to store DFPWM data in a WAV file.
+
+## Overview
+DFPWM data is specified through the `WAVE_FORMAT_EXTENDED` data type, with the UUID `3ac1fa38-811d-4361-a40d-ce53ca607cd1`. Channels are stored interleaved, and samples are packed in groups of 8 per byte, as is consistent with raw DFPWM data. Only DFPWM1a data is considered here - original DFPWM should not be stored using this UUID.
+
+## `fmt `&nbsp;chunk
+The `fmt `&nbsp;chunk follows the standard layout for `WAVE_FORMAT_EXTENDED`, using the assigned DFPWM UUID:
+
+| Offset | Bytes | Description                                     |
+|--------|-------|-------------------------------------------------|
+| 0x00   | 2     | Format code (`0xFFFE`)                          |
+| 0x02   | 2     | Number of channels                              |
+| 0x04   | 4     | Sample rate                                     |
+| 0x08   | 4     | Byte rate (sample rate * channels / 8)          |
+| 0x0C   | 2     | Block align (ceil(channels / 8))                |
+| 0x0E   | 2     | Bits per sample (1)                             |
+| 0x10   | 2     | Extra data size (`0x16`)                        |
+| 0x12   | 2     | Valid bits per sample (1)                       |
+| 0x14   | 4     | Channel configuraion                            |
+| 0x18   | 16    | DFPWM UUID (`3ac1fa38811d4361a40dce53ca607cd1`) |
+
+## `fact` chunk
+The `fact` chunk is required when using a non-PCM format. This is simply a 32-bit integer specifying the total number of samples in a single channel. This should be equal to (data length * 8 / channels).
```

Comment:
perhaps change the expression here to something like `(data_length * 8 / channels)`, code formatting included?